70_Pushover.pm

```diff
@@ -235,80 +245,100 @@ sub Pushover_CGI() {
         # search for existing receipt
         keys %{ $hash->{READINGS} };
         while ( my ( $key, $value ) = each %{ $hash->{READINGS} } ) {
-            $revReadings{ $value->{VAL} } = $1
-              if ( defined( $value->{VAL} ) && $key =~ /^cb_(\d+)$/ );
+            if ( defined( $value->{VAL} ) && $key =~ /^cb_(\d+)$/ ) {
+                $revReadings{ $value->{VAL} } = $1;
+                $revReadings{ $1 } = $1;
+            }
```

Comment:
6) Fixed a bug in the callback logic that made returned an error when trying to execute an action

$value->{VAL} is the receipt ID, but within the URL later on we will find the FhemCallbackId, which is $1.